tests/test_tartiflette_app.py

```diff
@@ -13,3 +16,23 @@ def test_path(engine: Engine):
 
     assert response.status_code == 200
     assert response.json() == {"data": {"hello": "Hello stranger"}}
+
+
+def test_tartiflette_app_as_sub_starlette_app(engine: Engine):
```

Comment:
Should we move this test to `test_mount.py`? This file is for testing the `TartifletteApp` class in isolation.

Reply:
ok, done !